requirements_docs.txt

```diff
@@ -1,3 +0,0 @@
-sphinx>=2.0.0
-sphinx-bluebrain-theme
-sphinx-autorun
```

Comment:
this is more a curiosity question. I see that in the CI https://neurodamus--22.org.readthedocs.build/en/22/

the theme is still bbp. However, we removed it from here. From the repo I cannot find anything about `bluebrain-theme`. Where does it pick it from?

Reply:
IMO, all the dependencies should be managed from one place when it's clear they can be (as opposed to requirements.txt) so I put them there:

https://github.com/openbraininstitute/neurodamus/pull/22/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R28

Reply:
strange that I could not find that line. Maybe I confused repos or branches as I was jugling with 3 branches in 2 repos. My bad. Thank you for the clarification